pyramid_oereb/views/webservice.py

```diff
@@ -44,7 +44,7 @@ def _is_json(self):
             bool: True if requested format is JSON.
         """
         url = self._request.current_route_url().split('?')
-        return url[0].endswith('.json')
+        return self._request.matchdict.get('format', '').lower() == 'json' or url[0].endswith('.json')
```

Comment:
I think this change is not related to the PR?

Reply:
I use this method for the creation of the `Parameter` object (e.g. line 162). I had to extend this method to respect also the format value of the request matchdict. The check of the URL ending only works for the old service URLs, where no format parameter has been defined in the specification.

Reply:
OK, thanks